web/source/dom/domEventHandlers.ts

```diff
@@ -602,8 +606,9 @@ namespace com.keyman.dom {
         osk._Show();
       }
       
-      // If clicked on DIV on the main element or on the scroller element, set caret to end of text
-      if(tTarg && tTarg == target || tTarg == scroller) {
+      // If clicked on DIV on the main element, rather than any part of the text representation,
+      // set caret to end of text
+      if(tTarg && tTarg == target) {
```

Comment:
What if `tTarg == scroller` now? Can that happen? (The previous condition was horrible with its mix of && and || but I'm not clear on the change here.)

Reply:
If `tTarg == scroller`, that means the click was within the text-containing area.  So, we handle that in the next 'rung' of the if-else sequence - the condition of `if(tTarg != scroller.nextSibling)` will be `true` in that case.

Reply:
Sure. But that means if `tTarg == scroller`, we are now running an entirely different piece of code which seems to be doing some quite different things, so not quite sure how that works out?

Reply:
Right.  That's a minor fix from the change made in https://github.com/keymanapp/keyman/pull/4638, as I actually shouldn't have included `scroller` in the check originally.